mantis-publish/mantis-publish-core/src/main/java/io/mantisrx/publish/internal/mql/MQLSubscription.java

```diff
@@ -117,8 +118,26 @@ public boolean matches(Event event) {
         }
     }
 
+	private List<String> sanitizeStreamSubjects(final List<String> subjects) {
+        return subjects.stream()
+                .map((String s) -> {
+                    if (s.toLowerCase().equals("observable") ||
+                            s.toLowerCase().equals("stream") ||
+							s.toLowerCase().equals("defaultstream")) {
```

Comment:
spacing if off

Reply:
Interesting. Looked the same in my editor but line 125 has spaces and 126 had tabs. I'm going to need to configure my editor to adhere to the formatting rules clearly.

Reply:
interesting that the formatter that we added recently isn't catching this issue?